### ISSUE TITLE
There is no way to define a column for default sorting 

### ISSUE THREAD

**Issue by rodrigoAbril** [MicroDroid/vue-materialize-datatable#26]
There is no way to define a column for default sorting

**Comment by MicroDroid** [comment]
This should be a good place for improvement.

For now, try ordering the dataset outside the plugin on initial render

**Comment by rodrigoAbril** [comment]
I have tried, but when the tables render there are always some rows out of order.
In my case the first one will always be the row with id 57,  but when i click to sort by id everyting fixes up....

I also noticed that the demo image in the documentation has the same error (first one is id 26)
![687474703a2f2f692e696d6775722e636f6d2f313574417058752e706e67](https://user-images.githubusercontent.com/847474/30883556-c75fd1c0-a2c9-11e7-9490-a9b142737ebc.png)

**Comment by justice47** [comment]
@rodrigoAbril I understand that your question was asked nearly 2 years ago, but here is temporary solution:
in DatatTable.vue find `sortColumn` data and change it to the column number you want your component to be sorted by on init.
@MicroDroid can we implement some new prop (:initSort=*column number* for example), which will change initial sorting?

**Comment by rodrigoAbril** [comment]
:) seems like a good solution

**Comment by MicroDroid** [comment]
@justice47 I'd rather find a better way since this is a controlled component and I'd rather not use local state with a prop like that.

**Comment by justice47** [comment]
@MicroDroid ok, if you will find a better way just post your thoughts about it, I can try to help you with that!

**Comment by justice47** [comment]
@MicroDroid maybe with approach will suit all needs:
Lets stay with ```currentPage``` in data and change it only once, then component is mounted.
Example:
props
```
props: {
 ...,
 initSortCol: {default: 0}
}
```
data
```
data: () => ({
 ...,
 sortColumn: 0,
}
```
mounted hook
```
mounted: function() {
 ...
 this.sortColumn = this.initSortCol
}
```
and then in component call
```
<datatable
 ...
 :initSortCol="2"
>
```

**Comment by MicroDroid** [comment]
Closed in #42
